apps/pie-storybook/stories/pie-notification.stories.ts

```diff
@@ -150,7 +150,6 @@ const Template : TemplateFunction<NotificationProps> = ({
         @pie-notification-close="${pieNotificationClose}"
         @pie-notification-open="${pieNotificationOpen}"
         >
-        <icon-alert-circle-filled slot="icon" size="s"></icon-alert-circle-filled>
```

Comment:
Does this mean you can remove the icon import at the top of this file?

Reply:
Removed!